_includes/parse-server/database.md

```diff
@@ -13,7 +13,7 @@ The Mongo requirements for Parse Server are:
 * MongoDB version 3.6
 * An SSL connection is recommended (but not required).
 
-If this is your first time setting up a production MongoDB instance, we recommend using either [mLab](http://www.mLab.com) or [ObjectRocket](https://objectrocket.com/). These are database-as-a-service companies which provide fully managed MongoDB instances, and can help you scale up as needed.
+If this is your first time setting up a production MongoDB instance, we recommend using [ObjectRocket](https://objectrocket.com/). This is a database-as-a-service company which provide fully managed MongoDB instances, and can help you scale up as needed.
```

Comment:
mLab has been bought by MongoDB, so the appropriate change would be to change the wording and link to MongoDB's Atlas free plan.

Reply:
Sure

Reply:
Sure.